### PR TITLE
added Issues and Pull Requests link to footer [Issue #263]

### PR DESCRIPTION
**Pull request by dalmia007**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [✓ ] Enhancement
- [ ] Documentation Update

## Description
Added Issues and Pull Requests link to footer

## Related Tickets & Documents
Issue 263

## QA Instructions, Screenshots, Recordings
Check the footer for links to Issues and Pull Requests page

## Added to documentation?

- [ ] readme

### CODE REVIEW

**Review by rahuldkjain — CHANGES_REQUESTED** [review]
Please remove changes of `package-lock.json` from this PR.

**Comment by dalmia007** [comment]
@rahuldkjain  I have tried to do it several times. However, git is still picking up that change in package-lock.json. It is not really a change. The file is still the same. I actually copied it from the master repo just to make sure its okay.

File: src/components/footer.js [hunk 1]
```diff
@@ -59,6 +59,24 @@ const Footer = () => {
               Releases
             </a>
           </div>
+          <div>
+            <a
+              href="https://github.com/rahuldkjain/github-profile-readme-generator/issues"
+              aria-label="Releases on Github rahuldkjain/github-profile-readme-generator"
```
Comment (rahuldkjain) [inline src/components/footer.js:65]: change aria-label to `Issues in rahuldkjain/github-profile-readme-generator`

File: src/components/footer.js [hunk 2]
```diff
@@ -59,6 +59,24 @@ const Footer = () => {
...
+              target="blank"
+            >
+              Issues
+            </a>
+          </div>
+          <div>
+            <a
+              href="https://github.com/rahuldkjain/github-profile-readme-generator/pulls"
+              aria-label="Releases on Github rahuldkjain/github-profile-readme-generator"
```
Comment (rahuldkjain) [inline src/components/footer.js:74]: change aria-label to `Pull Requests in rahuldkjain/github-profile-readme-generator`

**Review by rahuldkjain — CHANGES_REQUESTED** [review]
Please fix the comments.

**Comment by dalmia007** [comment]
@rahuldkjain the aria labels have been corrected

**Review by rahuldkjain — APPROVED** [review]
Superb @dalmia007.

**Pull request merged**